packages/web/src/components/organisms/EntityDetails/EntityDetailsHeader.tsx

```diff
@@ -64,6 +65,18 @@ const EntityDetailsHeader: FC<EntityDetailsHeaderProps> = ({
     });
   };
 
+  const entityOptionsMenu = useMemo(
+    () =>
+      [
+        {key: 1, label: <span onClick={onEditTest}>Edit Test</span>},
```

Comment:
`EntityDetailsHeader` is not only used for tests, so the prop should be called `onEdit` and this label should be based on entity type, not static `Edit Test`